Update community resources for migration guides (#11376)
withastro/docs: src/content/docs/en/guides/cms/directus.mdx

service: Directus
i18nReady: true
---
import { CardGrid, LinkCard } from '@astrojs/starlight/components';

[Directus](https://directus.io/) is a backend-as-a-service which can be used to host data and content for your Astro project.

## Official Resources

- [Getting Started with Directus and Astro](https://docs.directus.io/blog/getting-started-directus-astro.html).

## Community Resources 

<CardGrid>
  <LinkCard title="Using Directus CMS with Neon Postgres and Astro to build a blog" href="https://neon.tech/guides/directus-cms" />
</CardGrid>

:::note[Have a resource to share?]

> If you found (or made!) a helpful video or blog post about using Directus with Astro, edit this page and add it here!

Author: ```suggestion
If you found (or made!) a helpful video or blog post about using Directus with Astro, [add it to this list](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/cms/directus.mdx)!
```
